### ISSUE TITLE
no deprecation in log 

### ISSUE THREAD

**Issue by oceanBT** [symfony/symfony#35367]
**Symfony version(s) affected**: Upgrade from 4.3 to 4.4

**Description**  
Symfony does not log any deprecation in my log. 

**How to reproduce**  
Minimal Example (created with 4.3 skeleton and upgraded to 4.4):

```
git clone https://github.com/oceanBT/symfony-deprecation-log
cd symfony-deprecation-log
composer install
bin/console server:run

curl http://localhost:8000/test

tail -f var/log/*.log  # no deprecations got logged :/
```
Changed files: 
https://github.com/oceanBT/symfony-deprecation-log/blob/master/config/packages/dev/monolog.yaml (but also original config does not work) 

https://github.com/oceanBT/symfony-deprecation-log/blob/master/src/Controller/TestController.php#L26

.env:
APP_ENV=dev

**Comment by jsfgreen** [comment]
I believe this is related to the issue I just opened up: https://github.com/symfony/symfony/issues/36009

**Comment by xabbuh** [comment]
This has probably been fixed by #36342. Please report back if that's not the case.

**Comment by oceanBT** [comment]
I just tested it. If I run composer update on my minimal example, it works now! Thx for the fix!
